gnodeb/context/gnbcpue.go

```diff
@@ -15,16 +15,9 @@ import (
 )
 
 type GnbCpUe struct {
-	Supi        string
-	GnbUeNgapId int64
-	AmfUeNgapId int64
-	Amf         *GnbAmf
-	Gnb         *GNodeB
-
-	// TODO: Sync map is not needed as it is handled single threaded
-	GnbUpUes sync.Map
-
-	WaitGrp sync.WaitGroup
+	Supi string
+	Amf  *GnbAmf
+	Gnb  *GNodeB
```

Comment:
Why not putting the pointers together (including `Log *logrus.Entry`)? Also, probably we can remove the text `// logger`, the variable itself seems self-explanatory, what do you think?